src/communication/handler.ts

```diff
@@ -38,28 +38,20 @@ async function hashPinnedHandler (
     await storageProvider?.rm(job.fileHash)
     roomLogger.info(`File ${job.fileHash} for offer ${job.offerId} unpinned`)
   }
-
-  // Leave room if not active jobs for that offer
-  const pendingJobsForOffer = await UploadJob.count({ where: { offerId } })
```

Comment:
Hmmm without this we will stay forever connected to the Room though 🤔
It could stay here right?

Reply:
We had the same logic in the GC function, but here we have a chance to leave the channel faster

Reply:
Without waiting until GC is executed

Reply:
Probably i will revert this